devicetypes/Elko-Super-Thermostat.src

```diff
@@ -52,8 +54,8 @@ metadata {
             }
         }
 		
-        valueTile("power", "device.power", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
-			state "default", label:'${currentValue} Watt', backgroundColor:"#ffffff"
+        valueTile("energy", "device.energy", inactiveLabel: false, decoration: "flat", width: 2, height: 1)) {
```

Comment:
There is an extra ending ')' on this line 57. Remove it and your patch works on Smartthings Classic on my Hub2.

Reply:
Thanks, have you verified if the general power reporting works also?

Reply:
Energy is not reported on my devices. It shows 0.0 kWh in all rooms even after heating has been active on and off for a day. I would assume that this should be a counter of used kWh. 
Bear in mind that my devices are from early 2018 so they might not have a firmware version which reports this.

Reply:
rboy1’s patch also introduced a configuration item, available from within the classic app’s device. This seems to be the actual power used when the heating is on (hos many wats the floor is drawing from the regulator) I have not measured this value yet, but setting this value to 1000w causes way to high reporting values. Have you tried this?

Reply:
I noticed the same after reading the code. I also found that the default empty config value would result in the power usage being multiplied by zero.

I did change the formula in my local deployment and assumed that the reported power from the thermostat was an average wattage over the 400 seconds as this seems to be the delay configured. 
I have no way of validating the actual power usage, but the counter does at least not seem too unreasonable.

Reply:
Fixed, the settings should also be accessible from the new app now.